source/js/buyers-guide/product-tab.js

```diff
@@ -0,0 +1,33 @@
+export default () => {
+  document.querySelectorAll(`#product-tab`).forEach((element) => {
+    const allTabs = element.querySelectorAll("span[data-product-label]");
+
+    // First tab is active by default
+    allTabs[0].classList.remove("tw-bg-white", "tw-text-black");
+    allTabs[0].classList.add("tw-bg-black", "tw-text-white");
+
+    allTabs.forEach((tabButton) => {
+      tabButton.addEventListener("click", (event) => {
+        event.stopPropagation();
+        event.preventDefault();
+        // reset active tab stylings
+        for (var i = 0; i < allTabs.length; i++) {
+          allTabs[i].classList.remove("tw-bg-black", "tw-text-white");
+          allTabs[i].classList.add("tw-bg-white", "tw-text-black");
+        }
+
+        // set active tab styling
+        event.currentTarget.classList.remove("tw-bg-white", "tw-text-black");
+        event.currentTarget.classList.add("tw-bg-black", "tw-text-white");
```

Comment:
since this is the same as what we do on L6-7, let's make this a function call that can take an element as argument.